gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 06 09:39:22 GMT 2017
+#Mon Dec 11 14:21:26 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
Don't forget to update the wrapper scripts as well.

Reply:
Should that have happened automatically?  Hmm..

Reply:
Not if you just manually changed the version number here. 

If you run the update from the command line
`./gradlew wrapper --gradle-version=4.1 --distribution-type all`

Or add this helper task to the root `build.gradle`
```
task wrapper(type: Wrapper) {
  gradleVersion = "4.1"
  distributionType = "all"
}
```
and run it with `./gradlew wrapper`, the wrapper jar and script should get updated when you do that.

Reply:
Sweet, thanks. Did not know this was required.